cmd/teleportercmd/relayercmd/logs.go

```diff
@@ -182,34 +182,34 @@ func getLogSubnet(
 	} {
 		value, b := logMap[key].(string)
 		if b {
-			subnetName := blockchainIDToSubnetName[value]
-			if subnetName != "" {
-				return subnetName
+			blockchainName := blockchainIDToBlockchainName[value]
+			if blockchainName != "" {
+				return blockchainName
 			}
 		}
 	}
 	return ""
 }
 
-func getBlockchainIDToSubnetNameMap(network models.Network) (map[string]string, error) {
-	subnetNames, err := app.GetSubnetNamesOnNetwork(network)
+func getBlockchainIDToBlockchainNameMap(network models.Network) (map[string]string, error) {
+	blockchainNames, err := app.GetBlockchainNamesOnNetwork(network)
 	if err != nil {
 		return nil, err
 	}
-	blockchainIDToSubnetName := map[string]string{}
-	for _, subnetName := range subnetNames {
-		_, _, _, blockchainID, _, _, _, err := teleporter.GetSubnetParams(app, network, subnetName, false)
+	blockchainIDToBlockchainName := map[string]string{}
+	for _, blockchainName := range blockchainNames {
+		blockchainID, err := contract.GetBlockchainID(app, network, contract.ChainSpec{BlockchainName: blockchainName})
 		if err != nil {
 			return nil, err
 		}
-		blockchainIDToSubnetName[blockchainID.String()] = subnetName
+		blockchainIDToBlockchainName[blockchainID.String()] = blockchainName
 	}
-	_, _, _, blockchainID, _, _, _, err := teleporter.GetSubnetParams(app, network, "", true)
+	blockchainID, err := contract.GetBlockchainID(app, network, contract.ChainSpec{CChain: true})
 	if err != nil {
 		return nil, err
 	}
-	blockchainIDToSubnetName[blockchainID.String()] = "c-chain"
-	return blockchainIDToSubnetName, nil
+	blockchainIDToBlockchainName[blockchainID.String()] = "c-chain"
+	return blockchainIDToBlockchainName, nil
 }
 
 func logLevelToEmoji(logLevel string) (string, error) {
```

Comment:
should we move this function to utils?

Reply:
also I feel like this function should not return `err` just `""` for the incorrect loglevel

Reply:
it is better for errors to be propagated, level msg should always be an avago one, if not the case, 
it is also better to stop the process so we, or relayer people, can fix the issue.
also